src/randomizer.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const getRandomNumber = () => {
```

Comment:
number should consist of 4 **_different_** digits
![image](https://github.com/mate-academy/node_bulls-and-cows/assets/147749101/13b1ad5d-a160-41e8-9396-413008a86a00)